### PR TITLE
add change log for 6.3

### PR DESCRIPTION
**Pull request by bghimireamd**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,21 @@
...
+- [PReLU] Implement PReLU backward (experimental)
+
+### Fixed
+- Fixed stream not being set when calling hipMemsetAsync
+- Fixed memory leak issue caused by incorrect transpose in find 2.0 (#3285)
+- Fixed memcopy data race by replacing hipMemcpy with hipMemcpyWithStream
+## Perfomance
+- MI300 TunaNet Update: CK FWD and WRW Solvers Updated 
+
+## MIOpen 3.2.0 for ROCm 6.2.0
```
Comment (amd-jnovotny) [inline CHANGELOG.md]:
```suggestion
## MIOpen 3.3.0 for ROCm 6.3.0

### Added

* [RNN] LSTM fwd
* [Mha] Mask is added for Forward pass 
* [GLU] Gated Linear Unit (this is an experimental feature)
* [PReLU] Implemented PReLU backward (this is an experimental feature)

### Optimized

- MI300 TunaNet Update: CK FWD and WRW Solvers Updated 

### Resolved issues

- Fixed unset stream when calling `hipMemsetAsync`
- Fixed a memory leak issue caused by an incorrect transpose in find 2.0 (see issue #3285 on GitHub)
- Fixed a `memcopy` data race by replacing `hipMemcpy` with `hipMemcpyWithStream`


## MIOpen 3.2.0 for ROCm 6.2.0
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,21 @@
 
 Full documentation for MIOpen is available [here](https://rocm.docs.amd.com/projects/MIOpen/en/latest/)
 
-## MIOpen-3.2.0 for ROCm 6.2.0
+## MIOpen 3.3.0 for ROCm 6.3.0
+### Added
+- [RNN] LSTM fwd
```
Comment (amd-jnovotny) [inline CHANGELOG.md]: Should this be forward or forwarding? Or is fwd okay?

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,21 @@
...
+- [Mha] Mask is added for Forward pass 
+- [GLU] Gated Linear Unit (experimental)
+- [PReLU] Implement PReLU backward (experimental)
```
Comment (amd-jnovotny) [inline CHANGELOG.md]: I believe PReLU Backward is an official name, so Backward s/b capitalized. Can you confirm?

File: CHANGELOG.md (hunk 2 above)
Comment (bghimireamd) [reply CHANGELOG.md]: I think fwd is ok

**Review by amd-jnovotny — APPROVED** [review]
looks good

**Comment by countradooku** [comment]
when officially on windows

**Pull request merged**